torch_xla/csrc/ops/dynamic_ir.h

```diff
@@ -47,6 +47,9 @@ class SizeNode : public XlaNode, public torch::lazy::DimensionNode {
  private:
   size_t dim_ = 0;
   int64_t upper_bound_;
+  mutable bool dyanmic_value_computed_ = false;
```

Comment:
nit: typo dynamic

Reply:
good catch!

Reply:
And also nit: the "represent" at line 51